cmd/Makefile

```diff
@@ -86,6 +86,7 @@ obj-$(CONFIG_CMD_LICENSE) += license.o
 obj-y += load.o
 obj-$(CONFIG_CMD_LOG) += log.o
 obj-$(CONFIG_ID_EEPROM) += mac.o
+obj-$(CONFIG_CMD_MCU_COM) += mcu_com.o
```

Comment:
_General_: Should we prefix Leica-specific commands with `leica_`? This way it would be easier to find and port those later on...

Cc: @htho-lgs 
Cc: @grembeter

Reply:
Makes sense ...

Reply:
I modified it with `CONFIG_LEICA_CMD_MCU_COM`, is it what you meant as prefix?